nss-day-cohort-41/tabloidmvc-the-lobster-rolls
# Sf initial

# Description
Referring to tickets #19 
1. Showing list of all Comments on an individual post (accessed through button in post details page);
2. Comments list is in order of creation date (displaying Subject, Content, Author of Comment, Creation Date)
3. Title of post shows on top of Comments List
4 . Link back to post is available on Comments List

Referring to tickets #20
1. From details view page for a post, user can click add comment button and be directed to form creating new comment for that post (form should include subject and content)
2. user can save; creation time should be set as current date time and once saved should be directed to comments list page for that post

## Type of change

# Testing Instructions
1. Pull down sf-initial
2. run TabloidMVC
3. Log in with proper email from database
4. Click posts from menu and click details on that individual post in your list
5. Click Add Comment and fill out form; should be brought back to comments list after save
6. Click View Comments and you should be brought to comments list for that post
7. Can also click Back to post and it will bring you back to details page for that post

# Checklist:
